colin/core/checks/abstract_check.py

```diff
@@ -16,16 +16,42 @@
 import json
 
 from six import iteritems
+from colin.core.fmf_metadata_loader import get_fmf_from_class, set_fmf_metadata
 
 
 class AbstractCheck(object):
     name = None
+    message = ""
+    description = ""
+    reference_url = ""
+    tags = []
+    base_init_list = ["message", "description", "reference_url", "tags"]
+    init_list = []
 
-    def __init__(self, message, description, reference_url, tags):
-        self.message = message
-        self.description = description
-        self.reference_url = reference_url
-        self.tags = tags
+    def _fmf_meta_init(self, args, kwargs):
```

Comment:
shouldn't this be `*args, **kwargs`?

Reply:
could be, but I forwarded these list and dict directly from `__init__`, so that no need to dereferenced them.